### PR TITLE
Merge 2.0-dev into master

### PR DESCRIPTION
**Pull request by jerboaa**
I'd like to merge 2.0-dev into master. This does it. No new PRs should be opened for 2.0-dev as I intend to delete it at some point in the future. OK to push?

### CODE REVIEW

**Comment by tmds** [comment]
Now is fine. Or we wait till RH build bot turns green.

GitHub tips of the day (which you may know already):
You could create this PR within the s2i-dotnetcore repo by navigating to the 2.0-dev branch and clicking the 'pull request' button at the top right. It avoids the local branch and it auto-updates when additional commits are pushed.
And if we merge this now. Pending PRs for 2.0-dev can be changed to master using the edit button at the top right.

File: README.md [hunk 1]
```diff
@@ -30,6 +31,9 @@ $ sudo docker build -f Dockerfile.rhel7 -t dotnet/dotnetcore-10-rhel7 .
 Usage
 ---------------------------------
 
+For information about usage of Docker image for .NET Core 2.0,
+see [2.0 usage documentation](2.0/README.md).
```
Comment (aslicerh) [inline README.md]: This path is wrong. Do we want a link for both the build and the runtime?

Comment (omajid) [inline README.md]: I think so.

Comment (jerboaa) [inline README.md]: Thanks. Can I fix this post merge?

**Comment by jerboaa** [comment]
@tmds I'd like to merge now if possible. For the CI builder to turn green that'll take a while. Re github. I didn't know that. Maybe next time ;-)

File: README.md (hunk 1 above)
Comment (omajid) [inline README.md]: No objections from me.

**Comment by tmds** [comment]
Go ahead @jerboaa

File: README.md (hunk 1 above)
Comment (aslicerh) [inline README.md]: I'm fine with that.

**Pull request merged**